vpi/common/common_vpi.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
common_vpi.h contains a single function that helps the user to register VPI callbacks and it is used in all the VPI examples.

I can put it out of common. I don't think that it can be added to vpi_user.h since vpi_user.h should contain only VPI calls/definitions. It can be split in a different compilation unit, but since it contains only a very simple function, I do not see a benefit in doing it. Also, since this repo uses shell scripts and not makefiles to generate the examples, the function gets recompiled anyways in each example.

Reply:
Let's keep it as `vpi/common.c` and `vpi/common.h` for now, and I'll ask Tristan about providing a `vpi_user_ghdl.h`.